backend/api/api_deployment_views.py

```diff
@@ -119,9 +125,27 @@ def create(
             headers=headers,
         )
 
-
-def get_error_from_serializer(error_details: dict[str, Any]) -> Optional[str]:
-    error_key = next(iter(error_details))
-    # Get the first error message
-    error_message: str = f"{error_details[error_key][0]} : {error_key}"
-    return error_message
+    @action(detail=True, methods=["get"])
+    def download_postman_collection(
+        self, request: Request, pk: Optional[str] = None
+    ) -> Response:
+        """Custom action to fetch a single instance."""
+        instance = self.get_object()
+        active_api_key = ""
+        try:
+            api_key_inst = instance.apikey_set.filter(is_active=True).first()
```

Comment:
what is the apikey_set here? Is it the list of api keys of that API? can we use .get here?